nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const mealsArray = require("./data/meals");
+const reviewsArray = require("./data/reviews");
+const reservationArray = require("./data/reservations.json");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+// meals
+const mealsReviewsArray = mealsArray.map((mealitem) => {
+  mealitem.reviews = reviewsArray.filter(
+    (element) => mealitem.id === element.mealId
+  );
+  return mealitem;
+});
+
+app.get("/meals", async (request, response) => {
+  response.send(mealsReviewsArray);
+});
+// cheap-meal
+const cheapMealArray = mealsArray.filter((mealitem) => {
```

Comment:
you can use mealsReviewsArray to avoid filtering twice

Reply:
Thanks for giving a feedback.. :)